Cubical/Data/Sigma/Properties.agda

```diff
@@ -2,7 +2,7 @@
 
 Basic properties about Σ-types
 
-- Characterization of equality in Σ-types using transport ([pathSigma≡sigmaPath])
+- Characterization of equality in Σ-types using transport ([ΣPath≡pathΣ])
```

Comment:
Is this comment wrong now? It would be great if you could add a few more things to this comment as the file contains a lot of useful results